backend/satellite_tools/mgr-repo-metadata-signing

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
```

Comment:
Mostly because it requires invoking `gpg` and other cmd line utilities. Also the logic is not complex. So it felt more natural and quicker to do it with a shell script.

Reply:
It's true that in the end it will be you and your Squad maintaining this, still I feel like discouraging adding so much new shell script code.

Reply:
For such a simple thing as a script which just calls some other commands, bash is the best what you can use (my point of view) . Using python or java for it would make the thing over complex.
bash can be "maintained" by everybody in the team.

Only if you start to have complex algorythms a different language makes sense.
Just calling a command and react on the result is bash world.